aw_client/client.py

```diff
@@ -255,65 +255,44 @@ def __init__(self, client, dispatch_interval=0):
         # Buckets that will have events queued to them, will be created if they don't exist
         self._registered_buckets = []  # type: List[Bucket]
 
-        self._queue = queue.Queue()
+        self._attempt_reconnect_interval = 10
 
         # Setup failed queues file
         data_dir = get_data_dir("aw-client")
         queued_dir = os.path.join(data_dir, "queued")
         if not os.path.exists(queued_dir):
             os.makedirs(queued_dir)
-        self.queue_file = os.path.join(queued_dir, self.client.name + ".v{}.json".format(self.VERSION))
 
-    def _create_buckets(self):
+        persistqueue_path = os.path.join(queued_dir, self.client.name + ".v{}.persistqueue.sql".format(self.VERSION))
```

Comment:
.sql usually means that it contains a sql command rather than a sqlite db, should probably be just .db

Reply:
Good point, will fix.